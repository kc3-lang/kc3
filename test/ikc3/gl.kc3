quote require GL.Triangle
require GL.Triangle
quote require GL.Vec2
require GL.Vec2
quote require GL.Vec3
require GL.Vec3
quote require GL.Vertex
require GL.Vertex
quote %GL.Vec2{}
%GL.Vec2{}
quote %GL.Vec3{}
%GL.Vec3{}
quote %GL.Vertex{}
%GL.Vertex{}
quote %GL.Triangle{}
%GL.Triangle{}
