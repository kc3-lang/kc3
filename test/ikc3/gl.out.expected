%GL.Vec2{}
%GL.Vec2{x: (F32) 0.0,
         y: (F32) 0.0}
%GL.Vec3{}
%GL.Vec3{x: (F32) 0.0,
         y: (F32) 0.0,
         z: (F32) 0.0}
%GL.Vertex{}
%GL.Vertex{position: %GL.Vec3{x: (F32) 0.0,
                              y: (F32) 0.0,
                              z: (F32) 0.0},
           normal: %GL.Vec3{x: (F32) 0.0,
                            y: (F32) 0.0,
                            z: (F32) 0.0},
           tex_coord: %GL.Vec2{x: (F32) 0.0,
                               y: (F32) 0.0}}
%GL.Triangle{}
%GL.Triangle{a: (U32) 0,
             b: (U32) 1,
             c: (U32) 2}
